### ISSUE TITLE
Tensorflow pipeline evaluation

### ISSUE THREAD

**Issue by balinda-1995** [RasaHQ/rasa_nlu#1250]
**Issue**:
I am trying to compare two piplines: sklearn and the tensorflow embedding pipeline on my chatbot. I want to run` rasa_nlu.evaluate`:
```
python -m rasa_nlu.evaluate \
    --data data/examples/rasa/demo-rasa.json \
    --model `projects/default/model_20180323-145833
```

I didn't understand what's the data file (demo-rasa.json here) I should enter. Is it the same as my training data? which doesn't make much sense since I will be testing my models on the same data I trained them with. Or should I create another data file with different examples?

**Comment by marcomaisel** [comment]
Yes, you should create another data file with examples you haven't used for training.

If you don't have enough data you can evaluate your pipeline using cross-validation. Then you don't need extra data for the evaluation. More infos on cross-validation can be found in the documentation

**Comment by balinda-1995** [comment]
Thanks it worked! I got an F1 score of 0.81 for sklearn and 0.9 for the tensorflow pipeline.
I was expecting a larger gap, does that make sense?

**Comment by marcomaisel** [comment]
Great that it worked!

Why did you expect a larger gap? I can only tell you about my own experience so far. On my evaluation data set I get f1-scores of 0,935 for spacy/sklearn and 0,968 for the tensorflow pipeline.

**Comment by balinda-1995** [comment]
Great then!
I was just expecting a big gap because when I read [this article](https://medium.com/rasa-blog/supervised-word-vectors-from-scratch-in-rasa-nlu-6daf794efcd8) , in the part " Quick experiment" the gap was uge!
